typescript/infra/scripts/deploy.ts

```diff
@@ -33,8 +33,8 @@ import {
 
 async function main() {
   const { module, fork, environment } = await getArgsWithModuleAndFork().argv;
-  const config = await getEnvironmentConfig();
-  const multiProvider = await config.getMultiProvider();
+  const coreConfig = await getEnvironmentConfig();
```

Comment:
I don't love calling this coreConfig since it's not actually `CoreConfig` type. 

I believe it's CoreEnvironmentConfig type, which I'd prefer be renamed EnvironmentConfig